remix.init/README.md

```diff
@@ -0,0 +1,62 @@
+# Welcome to Remix!
+
+- [Remix Docs](https://remix.run/docs)
+- [Netlify Functions Overview](https://docs.netlify.com/functions/overview)
+
+## Netlify Setup
+
+1. Install the [Netlify CLI](https://www.netlify.com/products/dev/):
+
+```sh
+npm i -g netlify-cli
+```
+
+If you have previously installed the Netlify CLI, you should update it to the latest version:
+
+```sh
+npm i -g netlify-cli@latest
+```
+
+2. Sign up and log in to Netlify:
+
+```sh
+netlify login
+```
+
+3. Create a new site:
+
+```sh
+netlify init
+```
+
+## Development
+
+The Remix dev server starts your app in development mode, rebuilding assets on file changes. To start the Remix dev server:
+
+```sh
+netlify dev
+```
+
+Open up [http://localhost:3000](http://localhost:3000), and you should be ready to go!
+
+The Netlify CLI builds a production version of your Remix App Server and splits it into Netlify Functions that run locally. This includes any custom Netlify functions you've developed. The Netlify CLI runs all of this in its development mode.
+
+```sh
```

Comment:
Since this is the same command as on line 37, does this section mean they run this command again or is the paragraph above explaining that this command also builds a production version of the Remix App Server?

Reply:
This looks like a copy paste error from the Netlify Functions README that already existed. I'll adjust this. Good spot!